components/footer/src/index.tsx

```diff
@@ -169,7 +169,7 @@ Footer.WidthContainer = WidthContainer;
 
 Footer.defaultProps = {
   children: undefined,
-  container: WidthContainer,
+  container: Footer.WidthContainer,
```

Comment:
any reason this was changed?

Reply:
yes - this was a change that got made fairly recently, IIRC attempting to chase down a phantom bug with storyshots.

the previous version meant that the documentation inside storybook said that the default value for this prop was `WidthContainer`, but there's no `WidthContainer` component directly exported.  in reality a user would need to use `Footer.WidthContainer`, so this change (back) means that the documentation is now more correct